cmd/supernode/app/root.go

```diff
@@ -70,6 +70,9 @@ func setupFlags(cmd *cobra.Command, opt *Options) {
 	flagSet.IntVar(&opt.ListenPort, "port", opt.ListenPort,
 		"ListenPort is the port supernode server listens on")
 
+	flagSet.IntVar(&opt.FileServerPort, "file-server-port", opt.FileServerPort,
```

Comment:
We should unify the names of supernode's ports in the documents and codes.
In docs: `registerPort(8002)`, `downloadPort(8001)`, and here they're: `ListenPort(8002)`, `FileServerPort(8001)`. Which one do you prefer?

Reply:
I prefer keep the 8002 with `ListenPort` and use `downloadPort` for 8001. WDYT?